### ISSUE TITLE
Disables "editor.showUnused" in VSCode

### ISSUE THREAD

**Issue by facetious** [Microsoft/typescript-tslint-plugin#15]
# Description
When using this in VSCode (as per the instructions in the README), `"editor.showUnused": true` no longer dims unused imports.

**Comment by mjbvz** [comment]
I can't repo this:

![screen shot 2018-10-29 at 2 42 08 pm](https://user-images.githubusercontent.com/12821956/47682182-d9bede80-db88-11e8-8c48-e1f1b242c932.png)

Please provide more information, such as sample code and os

**Comment by facetious** [comment]
@mjbvz It looks like maybe I wasn't clear enough about the issue. I have `showUnused: true` in my configuration.

# Without this Plugin
<img width="799" alt="screen shot 2018-10-30 at 8 51 56 am" src="https://user-images.githubusercontent.com/1058563/47719318-25bd6200-dc21-11e8-9b84-a351f0e81e95.png">

# With this Plugin
<img width="776" alt="screen shot 2018-10-30 at 8 50 49 am" src="https://user-images.githubusercontent.com/1058563/47719319-25bd6200-dc21-11e8-9bf0-980ffac0fb12.png">

As you can see, the `OnDestroy` and `OnInit` unused imports are no longer dimmed when the plugin is enabled.

**Comment by mjbvz** [comment]
Yes this is what I tested. Please provide more information about your project: `tsconfig.json`, `tslint.json`, example code, ...

**Comment by facetious** [comment]
@mjbvz It looks to me like you have tested unused variables, not imports. You have also included an unrelated tslint error in your screenshot.

I have updated the body of the original.

**Comment by mjbvz** [comment]
@facetious Thanks. Also happens with imports. They are implemented the same way. The screenshots includes the error to show that tslint is enabled

**Comment by mjbvz** [comment]
Still can't repo this: ![screen shot 2018-10-31 at 12 55 06 pm](https://user-images.githubusercontent.com/12821956/47814925-3696d200-dd0c-11e8-93eb-41207d13c348.png)

- Are you using a workspace version of typescript?
- What os?

**Comment by facetious** [comment]
I am using TypeScript 2.8.4 on Mac (macOS 10.13.6).
